tools/python/boutiques/tests/test_export.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+import os
+from unittest import TestCase
+from boutiques import bosh
+from boutiques import __file__ as bfile
+
+
+class TestImport(TestCase):
+    def get_examples_dir(self):
+        return os.path.join(os.path.dirname(bfile),
+                            "schema", "examples")
+
+    def test_export(self):
```

Comment:
Do we have an example for what this descriptor should look like in CARMIN format? If so, we should also have an assertion that they are equal. No need to hold up the PR for this, but comment if you agree and we'll make an issue.

Reply:
We don't at the moment. I haven't checked that this passes CARMIN validation. Counting on @louis-ver and @simon-dube to clarify that. See https://github.com/boutiques/boutiques/issues/222